config/rbac/isbservicerollout_editor_role.yaml

```diff
@@ -0,0 +1,27 @@
+# permissions for end users to edit isbservicerollouts.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    app.kubernetes.io/name: numaplane
+    app.kubernetes.io/managed-by: kustomize
+  name: isbservicerollout-editor-role
+rules:
+- apiGroups:
+  - numaplane.numaproj.io.github.com.numaproj-labs
```

Comment:
Are we still using this group?

Reply:
Good catch! No, we should be using `numaplane.numaproj.io`. This could be an issue with kubebuilder.